server/settings.gradle.kts

```diff
@@ -0,0 +1,43 @@
+import global.genesis.gradle.plugin.simple.ProjectType
+
+rootProject.name = "genesisproduct-{{appName}}"
+
+pluginManagement {
```

Comment:
double pluginInManagement, I believe this was fixed in the main app seed, double check there.